extra/docs_landing.html

```diff
@@ -79,14 +79,23 @@
     text-align: center;
   }
 
+  #guide #viewall h2 {
+    font-size: 40px !important;
+    font-weight: 600 !important;
+    margin: 16px 0 !important;
+  }
+
   #guide h2 {
-    font-size: 40px;
+    text-align: left;
+    font-size: 20px;
+    margin: 16px 16px;
     font-weight: 500;
-    margin: 16px 0;
   }
   #guide h3 {
-    margin: 0;
-    font-weight: 400;
+    text-align: left;
+    font-size: 20px;
+    margin: 16px 16px;
+    font-weight: 500;
```

Comment:
@reakaleek this is my attempt at fixing the weird headings happening on the landing page:

<img width="1309" alt="Screenshot 2025-04-11 at 10 33 00 AM" src="https://github.com/user-attachments/assets/1a12068e-dc31-45aa-8f61-6f2ba86df423" />